clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/CreateFanClub.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { SafeAreaView, StyleSheet } from 'react-native';
+import { Box, Button, Text } from 'native-base';
+import { useRoute } from '@react-navigation/native';
+import Swiper from 'react-native-swiper';
+import { useTheme } from 'themes';
+import I18n from 'i18n/i18n';
+import Icon from 'views/components/base/Icon';
+import { Icons } from 'assets/icons';
+import useNavigationWithParams from 'utils/hooks/useNavigationWithParams';
+import DetailsContainer from './containers/DetailsContainer';
+import VisualContainer from './containers/VisualContainer';
+import PrivacyContainer from './containers/PrivacyContainer';
+import { fanClubsStore } from '../../store';
+import { ICreateFanClubProps } from './types';
+
+const CreateFanClub: React.FC<ICreateFanClubProps> = () => {
+  const { colors } = useTheme();
+  const { goBack } = useNavigationWithParams();
+  const route = useRoute();
+
+  const { add } = fanClubsStore.useSelector('add');
+
+  const slides = React.useMemo(() => {
+    return [<DetailsContainer />, <VisualContainer />, <PrivacyContainer />];
+  }, []);
+
+  const [index, setIndex] = React.useState(0);
+  const refSwiper = React.useRef<Swiper | null>(null);
+
+  const isFirstAction = index === 0;
+  const isLastAction = index === slides.length - 1;
+
+  // open component by index - when index was changed
+  React.useEffect(() => {
+    refSwiper.current?.scrollTo(index, true);
+  }, [index]);
+
+  // decrease index
+  const onPrevPress = React.useCallback(() => {
+    setIndex((oldIndex: number) => {
+      return oldIndex === 0 ? oldIndex : oldIndex - 1;
+    });
+  }, []);
+
+  // increase index
+  const onNextPress = React.useCallback(() => {
+    if (isLastAction) {
+      return fanClubsStore.create();
+    }
+
+    setIndex((oldIndex: number) => {
+      return oldIndex === slides.length - 1 ? oldIndex : oldIndex + 1;
+    });
+  }, [isLastAction, slides.length]);
+
+  const isSubmitEnabled = React.useMemo(() => {
+    if (!isLastAction) {
+      return true;
+    }
+
+    return add.valid;
+  }, [add.valid, isLastAction]);
+
+  React.useEffect(() => {
+    if (!route.params?.selected) {
+      return;
+    }
+
+    if (route.params?.selected.dataType === 'team') {
+      fanClubsStore.setAddFieldValue('teamId', route.params.selected.id);
+    }
+  }, [route.params]);
+
+  React.useEffect(() => {
+    if (add.status === 'success') {
+      fanClubsStore.reset();
+      goBack();
+
+      // @TODO: show success message
```

Comment:
let's push to new fan club screen

Reply:
will add with new PR, and we need to do same think with create event.